js/core/include-config.js

```diff
@@ -0,0 +1,20 @@
+// Module core/include-config
+// Inject's the document's configuration into the head as JSON.
+
+define(
+  [],
+  function() {
+    return {
+      run: function(conf, doc, cb, msg) {
+        msg.pub("start", "core/include-config");
+        var script = doc.createElement("script");
+        script.id = "respecConfig";
+        script.innerHTML = JSON.stringify(conf, null, 2);
+        script.type = "application/json";
```

Comment:
Is the above validator friendly? I suppose so, but just want to make sure, here.